Task/Pages/search_results_page.py

```diff
@@ -0,0 +1,12 @@
+from playwright.sync_api import Page
+
+class SearchResultsPage:
+    def __init__(self, page: Page):
+        self.page = page
+
+    def search_song(self, song_name: str):
+        return self.page.locator(f"//div[contains(@class, 'encore-text') and contains(text(), '{song_name}')]")
+
+    def check_song_exists(self, song_name: str) -> bool:
```

Comment:
better to name methods that returns bool starting with is/has/does/... verbs, like does_song_exist or smth similar